release-notes.md

```diff
@@ -1,5 +1,15 @@
 # Release notes
 
+## October 7, 2019
+
+The following updates were made in this release:
+
+**iOS Mobile Services 1.0.3**
+
+* Fullscreen messages now use WKWebViews from WebKit.framework.
+* Local notifications now use UserNotifications framework.
+* Fixed a crash issue.
```

Comment:
Any additional information we can provide? what happened when the crash occurred? where did the crash occur?